src/Auxillary/json_schema/schema_generation.cpp

```diff
@@ -20,7 +21,12 @@ namespace Aux::schema {
   ///////////////////////////////////////////////////////////////////////////
 
   int make_full_factory_schemas(path grazer_dir) {
-    Model::Componentfactory::Full_factory full_factory;
+    nlohmann::json defaults = aux_json::get_json_from_file_path(
+                                  grazer_dir / "problem" / "problem_data.json")
+                                  .at("problem_data")
+                                  .at("Network_problem")
+                                  .value("defaults", R"({})"_json);
+    Model::Componentfactory::Full_factory full_factory(defaults);
```

Comment:
This is the reason the key error "Network_problem" is happening. I am assuming that a "problem_data" json with defaults exists right now and forgot that I wanted to make this more graceful. What do you think about a simply try and catch block around this? I.e. catch if the file does not exist, catch keyerrors

Reply:
Although I am surprised that the basetest example is causing this problem... Since that should have the right file